### ISSUE TITLE
0.15.0 new session response value does not match spec

### ISSUE THREAD

**Issue by jleyba** [mozilla/geckodriver#529]
For a new session response, geckodriver 0.15.0 returns `{value: {sessionId: string, value: {}}`. The spec indicates this should be returned as `{value: {sessionId: string, capabilities: {}}`.

From the spec:

>  <ol start=11><li><p>Let <var>body</var> be a JSON <a>Object</a> initialised with:
>  <dl>
>  <dt><code>sessionId</code>
>  <dd><var>session id</var>
>  <dt><code>capabilities</code>
>  <dd><var>capabilities result</var>
>  </dl>
>  </ol>
>  ...
>  <ol start=14><li><p>Return <a>success</a> with data <var>body</var>.</ol>

**Comment by andreastt** [comment]
Bug is in https://github.com/mozilla/webdriver-rust/blob/master/src/response.rs#L79.

**Comment by andreastt** [comment]
This will be fixed as part of https://github.com/mozilla/webdriver-rust/pull/67/files#diff-6cfc163228cc31bed4a7214fa0b69300R69.

**Comment by andreastt** [comment]
This is fixed in webdriver 0.24.0 and https://github.com/mozilla/geckodriver/issues/462 onwards.

**Comment by nbuso** [comment]
Hi just tried with the version 0.16 firefox 53 and it's generating the same error:

```
[09:42:46] E/launcher - WebDriverError: Unable to parse new session response: {"value": {"sessionId":"3f3b551c-8c99-40bf-9798-872897d06151","capabilities":{"acceptInsecureCerts":false,"browserName":"firefox","browserVersion":"53.0","moz:accessibilityChecks":false,"moz:processID":29958,"moz:profile":"/tmp/rust_mozprofile.TMt7uBuhi83x","pageLoadStrategy":"normal","platformName":"linux","platformVersion":"4.10.10-200.fc25.x86_64","rotatable":false,"specificationLevel":0,"timeouts":{"implicit":0,"pageLoad":300000,"script":30000}}}}
    at WebDriverError (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/selenium-webdriver/lib/error.js:27:5)
    at doSend.then.response (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/selenium-webdriver/lib/http.js:445:19)
    at process._tickCallback (internal/process/next_tick.js:103:7)
From: Task: WebDriver.createSession()
    at Function.createSession (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/selenium-webdriver/lib/webdriver.js:777:24)
    at Function.createSession (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/selenium-webdriver/firefox/index.js:640:55)
    at Direct.getNewDriver (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/lib/driverProviders/direct.ts:112:25)
    at Runner.createBrowser (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/lib/runner.ts:225:39)
    at q.then.then (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/lib/runner.ts:391:27)
    at _fulfilled (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/node_modules/q/q.js:796:13)
    at /home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/node_modules/q/q.js:556:49
    at runSingle (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/node_modules/q/q.js:137:13)
    at flush (/home/nbuso/projects/ebeye-local/src/ebinocle-webjs/node_modules/protractor/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
[09:42:46] E/launcher - Process exited with error code 199
```

```
$ ./node_modules/protractor/bin/webdriver-manager status
[09:48:41] I/status - selenium standalone version available: 3.4.0 [last]
[09:48:41] I/status - chromedriver version available: 2.29 [last]
[09:48:41] I/status - geckodriver version available: v0.16.0 [last]
[09:48:41] I/status - android-sdk is not present
[09:48:41] I/status - appium is not present
```

Am I missing any configuraton?
